flatcar-linux.json

```diff
@@ -0,0 +1,65 @@
+{
+    "variables": {
+        "flatcar_channel": "alpha",
+        "flatcar_version": "1702.1.0",
+        "iso_checksum": "",
+        "iso_checksum_type": "none",
+        "install_target": "/dev/vda",
+        "disk_size": "40000",
+        "memory": "2048M",
+        "boot_wait": "45s",
+        "ignition": "ignition.json",
```

Comment:
@dongsupark is there a possibility that the path to `ignition.json` will ever change? We are manually generating it in the Makefile so we already know what our ignition file will be. I'm thinking of removing this variable and explicitly stating the file name where it is being used.

Reply:
@kosyfrances Yeah, maybe it can be removed. Just try removing it, and see if it works. :-)

Reply:
I removed it and just added the file name where it was needed and it worked fine.